packages/shopify-codemod/transforms/remove-empty-catch-blocks.js

```diff
@@ -0,0 +1,16 @@
+export default function removeEmptyCatchBlocks({source}, {jscodeshift: j}, {printOptions = {}}) {
+
+  return j(source)
+    .find(j.TryStatement, {
+      type: 'TryStatement',
+      handler: {
+        type: 'CatchClause',
```

Comment:
Can you use `j.TryStatement.name`/ `j.CatchClause.name` instead of the strings? I know we are inconsistent about this but I'd like to stick to that style.